.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,26 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: 
+    - "v*"
+    - '!*dev*'
+    - '!*pre*'
+    - '!*post*'
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+
+  # Release to PyPI after a RELEASE gets PUBLISHED in @spacetelescope/astrocut/main
+  publish-to-pypi:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    name: Publish to PyPI
```

Comment:
I'm not sure 'name' is supported here?

Reply:
I have the GitHub Actions extension installed on VSCode and it underlines things that are not supported/formatted correctly. This `name` arg isn't underlined, and it also worked fine when I was running the Test PyPI jobs.